### ISSUE TITLE
Disabling False Positives within the source code

### ISSUE THREAD

**Issue by v1v** [willthames/ansible-lint#29]
Use Case:

As a developer I would like to have a way of removing false positives within the source code rather than a global configuration

For example:

```yml
- name
  lineinfile: dest=/tmp/file.properties regexp="^Listen " line="something ${value}"
```

```
[ANSIBLE0001] Old style (${var}) brackets
examples/example.yml:2
```

I would like something like the below snippet:

```yml
# ansible-lint:disable
- name
  lineinfile: dest=/tmp/file.properties regexp="^Listen " line="something ${value}"
[...]
# ansible-lint:disable ANSIBLE0001
- name
  lineinfile: dest=/tmp/file.properties regexp="^Listen " line="something ${value}"
```

This feature is already part of the rubucop (ruby static analyzer): https://github.com/bbatsov/rubocop#disabling-cops-within-source-code
